### PR TITLE
feature/refacto : refacto DataBaseContext, FixturesContext

### PR DESCRIPTION
**Pull request by MLKiiwy**

### CODE REVIEW

File: Behat/Context/DatabaseContext.php [hunk 1]
```diff
@@ -297,7 +297,7 @@ public function assertIHaveSqlQueryResult($count)
      *
      * @When the SQL query result row :rowNumber should contain values:
      */
-    public function sqlQueryResultShouldContainValues($rowNumber, PyStringNode $stringNode)
+    public function sqlQueryResultShouldContainValuesAtRow($rowNumber, PyStringNode $stringNode)
```
Comment (vincentchalamon) [inline Behat/Context/DatabaseContext.php]: Missing `row` parameter ?

File: Behat/Context/FixturesContext.php [hunk 2]
```diff
@@ -62,4 +58,18 @@ public function purgeDatabase($connectionName)
...
     }
+
+    /**
+     * @param TableNode $tableNode
+     * @return array
+     */
+    protected function tableNodeToArray(TableNode $tableNode)
+    {
+        $array = [];
+        foreach ($tableNode->getRows() as $row) {
```
Comment (vincentchalamon) [inline Behat/Context/FixturesContext.php]:
```php
array_map
```

File: Behat/Context/FixturesContext.php [hunk 3]
```diff
@@ -62,4 +58,19 @@ public function purgeDatabase($connectionName)
...
     }
+
+    /**
+     * @param TableNode $tableNode
+     *
+     * @return array
+     */
+    protected function tableNodeToArray(TableNode $tableNode)
+    {
+        return array_map(
```
Comment (vincentchalamon) [inline Behat/Context/FixturesContext.php]:
```php
return array_map(function ($row) {
    return $row[0];
}, $tableNode->getRows());
```

**Comment by guillaumevoisin** [comment]
:+1:

**Comment by vincentchalamon** [comment]
:+1:

**Pull request merged**